tests/Client/CliTests.cs

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Diagnostics;
+using System.Reflection;
+using AzureMcp.Services.Azure;
+using AzureMcp.Tests.Client.Helpers;
+using Xunit;
+
+namespace AzureMcp.Tests.Client;
+
+public class NpxTests : IClassFixture<LiveTestSettingsFixture>
+{
+    private readonly LiveTestSettings _settings;
+
+    public NpxTests(LiveTestSettingsFixture fixture)
+    {
+        _settings = fixture.Settings;
+
+        if (string.IsNullOrEmpty(_settings.TestPackage))
+        {
+            Assert.Skip("Can only test packages ");
+        }
+    }
+
+    [Fact]
+    [Trait("Category", "Live")]
+    public async Task Help_command_should_return_help()
+    {
+        var outputLines = await RunCommand("--help");
+        var concatenatedOutput = string.Join(Environment.NewLine, outputLines.Output);
+        Assert.NotEmpty(concatenatedOutput);
+        Assert.Contains("azmcp [command] [options]", concatenatedOutput);
+    }
+
+    private async Task<(string[] Output, string[] Error, int ExitCode)> RunCommand(params string[] arguments)
```

Comment:
Is this common in C#? record vs tuple?

Reply:
Named tuple is fairly new in C#.  It's probably not super common yet.

Async method can't have output parameters, so getting multiple output from an async method requires some compound type (struct,class,etc).  The named tuple is the least amount of overhead to get some deconstructing call where the consumer and provider have a clear definition of the data in each field.

i.e. my producer declares a shape with named parts and my consumer can access the named parts with their static types and I don't have to define any additional type to accommodate the single method call.

Reply:
I wouldn't use a named tuple in the public api as its names are all syntactic sugar and would be lost post-compile.